e2e/autoscript-apps/project.json

```diff
@@ -29,14 +30,21 @@
     },
     "serve": {
       "executor": "@nrwl/web:dev-server",
-      "defaultConfiguration": "development",
+      "defaultConfiguration": "production",
```

Comment:
Why are we choosing production as the default config here?

Reply:
I am chosing production as default so that the "build" that we run in precommits doesn't default to starting the "watcher", but just "builds". 

The config for development is what would spawn the `start:e2e` command, so that by default will spawn in "watch" mode.